### PR TITLE
Added mailman_cesnet_user_mail service

### PR DESCRIPTION
**Pull request by zlamalp**
- We want mailman_cesnet service version, which takes user
  preferred mail instead of member mail.

### CODE REVIEW

File: gen/mailman_cesnet_user_mail [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+		if($useLangVariants) {
+			my $userLangAttribute = $data->getUserAttributeValue( member => $memberId, attrName => $A_USER_LANG );
+			if(defined($userLangAttribute) && $userLangAttribute eq "cs") {
+				$listNameWithLang = $listNameCS;
+			} else {
+				$listNameWithLang = $listNameEN;
+			}
+		}
+
+		my $userMailAttribute = $data->getMemberAttributeValue( member => $memberId, attrName => $A_USER_MAIL );
```
Comment (stavamichal) [inline gen/mailman_cesnet_user_mail:91]: You need to use "getUserAttributeValue" here. Parameters are the same.

File: slave/process-mailman-cesnet-user-mail/changelog [hunk 2]
```diff
@@ -0,0 +1,6 @@
+perun-slave-process-mailman-cesnet-user-mail (3.0.0) stable; urgency=low
+
+  * Copied implementation of perun-slave-process-mailman-cesnet
```
Comment (stavamichal) [inline slave/process-mailman-cesnet-user-mail/changelog:3]: Probably would be great to have also default purpose of this service even if it is just a copy of the existing service.

**Review by stavamichal — CHANGES_REQUESTED** [review]

File: gen/mailman_cesnet_user_mail [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+# Generates members of a mailing list #
+#######################################
+
+use strict;
+use warnings;
+use perunServicesInit;
+use perunServicesUtils;
+use Text::Unidecode;
+use POSIX qw/strftime/;
+use Data::Dumper;
```
[the remark this answers is not in the document]
Comment (zlamalp) [reply gen/mailman_cesnet_user_mail]: Fixed.

File: gen/mailman_cesnet_user_mail (hunk 1 above)
Comment (zlamalp) [reply gen/mailman_cesnet_user_mail]: Fixed.

File: slave/process-mailman-cesnet-user-mail/changelog (hunk 2 above)
Comment (zlamalp) [reply slave/process-mailman-cesnet-user-mail/changelog:4]: More info added.

**Review by stavamichal — APPROVED** [review]

**Pull request merged**